glean-core/android/build.gradle

```diff
@@ -132,37 +141,24 @@ afterEvaluate {
 }
 
 configurations {
-    // There's an interaction between Gradle's resolution of dependencies with different types
-    // (@jar, @aar) for `implementation` and `testImplementation` and with Android Studio's built-in
-    // JUnit test runner.  The runtime classpath in the built-in JUnit test runner gets the
-    // dependency from the `implementation`, which is type @aar, and therefore the JNA dependency
-    // doesn't provide the JNI dispatch libraries in the correct Java resource directories.  I think
-    // what's happening is that @aar type in `implementation` resolves to the @jar type in
-    // `testImplementation`, and that it wins the dependency resolution battle.
-    //
-    // A workaround is to add a new configuration which depends on the @jar type and to reference
-    // the underlying JAR file directly in `testImplementation`.  This JAR file doesn't resolve to
-    // the @aar type in `implementation`.  This works when invoked via `gradle`, but also sets the
-    // correct runtime classpath when invoked with Android Studio's built-in JUnit test runner.
-    // Success!
-    jnaForTest
+    // Configurations are a somewhat mysterious Gradle concept.  For our purposes, we can treat them
+    // sets of files produced by one component and consumed by another.
```

Comment:
I'm a little confused by the clash between "we can treat them as files produced by one component and _consumed_ by another" and the fact that we set `canBeConsumed = false`. Can you clarify this in the comment or is this the "mysterious" part?

Reply:
Copy-pasta! Quite honestly I have no idea. I copied that from a-s. Might as well remove those comments to avoid more confusion.